packages/components/pagination/package.json

```diff
@@ -44,6 +44,8 @@
   "homepage": "https://sparkui.vercel.app",
   "license": "MIT",
   "dependencies": {
+    "@spark-ui/button": "^6.2.9",
+    "@spark-ui/icon-button": "^6.2.9",
```

Comment:
I don't think this should be included in this PR

Reply:
Just removed these changes. I may open a PR for these ones :+1: